jerry-libm/scalbn.c

```diff
@@ -33,14 +33,15 @@
  * exponentiation or a multiplication.
  */
 
-#define two54  1.80143985094819840000e+16 /* 0x43500000, 0x00000000 */
 #define twom54 5.55111512312578270212e-17 /* 0x3C900000, 0x00000000 */
```

Comment:
Please make sure that the `#define` -> `const double` rewrite is applied consistently, i.e., no constants remain `#define`d.